src/db_options.rs

```diff
@@ -150,6 +150,40 @@ impl BlockBasedOptions {
             );
         }
     }
+
+    /// Format version, reserved for backward compatibility.
+    /// See https://github.com/facebook/rocksdb/blob/f059c7d9b96300091e07429a60f4ad55dac84859/include/rocksdb/table.h#L249-L274.
+    ///
+    /// Default: 2.
+    pub fn set_format_version(&mut self, version: i32) {
```

Comment:
not sure what type should be exposed in the rust API

Reply:
Other methods seem to use `i32` when the C-function uses `int` so seems fine to me?